extensions/binding/org.eclipse.smarthome.binding.weatherunderground/OSGI-INF/WeatherUndergroundDiscoveryService.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was referring to delaying the docs. anyway, here you go :-) https://github.com/eclipse/smarthome/pull/4002